metadata_transpiler_service/creation_models.py

```diff
@@ -638,6 +638,14 @@ class CreatePublication(InformationContentEntity):
         None,
         description="""The study abstract that describes the goals. Can also hold abstract from a publication related to this study.""",
     )
+    author: Optional[str] = Field(
```

Comment:
Are the creation model changes supposed to come from YAML artifacts? Does this addition require a change in the YAML?

Reply:
Yes, they are.